.github/workflows/cd.yml

```diff
@@ -7,6 +7,10 @@ on:
         description: "preid name(alpha, beta, rc, stable)"
         required: true
         default: "rc"
+      isPreview:
+        description: "ensure release ttk for preview(yes or no)"
+        required: false
```

Comment:
Nit: required = true?

Reply:
I don't think it's required, since there may rc, stable release. 
they may not care about this input